capstone/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -4,6 +4,7 @@
 class Constants {
   private Constants() {}
 
+  static final String CLUB_PROP = "Club";
```

Comment:
I think we already have a "club" and "clubs" prop, so if possible you could just use that one? it's in a more recent commit though, so guess this is something we would fix in a future push

Reply:
I just realized this is from my code lmao thanks for catching this. Should we just use the constant "club" instead or does this warrant "Club" because its specifically for the entity kind?

Reply:
Haha we probably shouldn't have both...I don't think I can see the other one you're talking about though, so maybe we can fix in the future?

Reply:
Unsure if this is what Megan meant but I see you have 

static final String PROPERTY_CLUBS = "clubs";

below which is pretty similar to "Club" defined here. If it's absolutely necessary to have the two very similar props feel free to keep. Otherwise it would be great to consolidate.

Reply:
@meghakak @mes712 I don't think any of my code uses the constant PROPERTY_CLUB = "clubs", so if neither of you guys use it either, feel free to remove it